hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestAbfsInputStreamReadFooter.java

```diff
@@ -0,0 +1,199 @@
+/**
```

Comment:
Run read tests in particular and contract test on a whole multiple times to check for any random failures.

Reply:
Yes, if 100 MB is set as buffer size, current logic is forcing a single server read request for 100 MB.

Reply:
Sure